scripting/ttt_core.sma

```diff
@@ -489,7 +489,7 @@ public Show_All()
 
 public Ham_TakeDamage_pre(victim, inflictor, attacker, Float:damage, bits)
 {
-  if(!my_return_check(attacker))
+  if(!my_return_check(attacker) && !(bits & DMG_BLAST)) // Exclude C4
```

Comment:
I need to rework this so that karma modifier is still being applied.

Reply:
Sure, sounds good